wami/app.js

```diff
@@ -414,6 +432,208 @@ viewImagesButton.addEventListener('click', async e => {
   }
 });
 
+// Process share target data when the app is launched
+async function processShareTargetData() {
+  // Check if we were launched via share target
+  const urlParams = new URLSearchParams(window.location.search);
+  const isShared = urlParams.get('share') === 'true';
+  
+  if (isShared) {
```

Comment:
When almost the entire body of a function is wrapped in an `if` statement like here, it's nice to reverse the condition and do an early return. This avoids having to indent the entire function body.
So, instead of `if (something) { .... the rest of the function .... }`, can you please do `if(not something) { return; } ... then the rest of the function, unindented` ?